internal/services/contracts_events_consumer.go

```diff
@@ -137,7 +137,9 @@ func (c *ContractsEventsConsumer) setPrivilegeHandler(e *ContractEventData) erro
 
 	nftCols := models.NFTPrivilegeColumns
 
-	err = udp.Upsert(context.Background(), c.db.DBS().Writer, true, []string{nftCols.ContractAddress, nftCols.TokenID, nftCols.Privilege, nftCols.UserAddress}, boil.Infer(), boil.Infer())
+	err = udp.Upsert(context.Background(), c.db.DBS().Writer, true,
+		[]string{nftCols.ContractAddress, nftCols.TokenID, nftCols.Privilege, nftCols.UserAddress},
+		boil.Whitelist(nftCols.UpdatedAt), boil.Infer())
```

Comment:
Wouldn't we also want to update expiry? I'm a little confused about how this is working.

Reply:
I think I'm convinced that this needs to be expanded to include `nftCols.Expiry`.

Reply:
You changed the whitelist! But I think the test is still not checking anything meaningful with regard to the expiry. Let’s talk in the morning.